src/ui.js

```diff
@@ -32,8 +32,10 @@ const controlPanelImg = document.getElementById('controlPanelImg');
 const cameraRefresh1 = document.getElementById('camera1-refresh');
 const cameraRefresh2 = document.getElementById('camera2-refresh');
 const ballsIndicatorBar = document.getElementsByClassName("balls-bar");
-const messageButton = document.getElementById("message-button")
-const messageText = document.getElementById("message-text")
+const messageButton = document.getElementById("message-button");
```

Comment:
Small issue, but I notice that there's not consistent use of single or double quotes—not a problem specifically with this PR but just in general make sure new code conforms to code style. Great job though, this is really just a nitpick.